app/models/notification.rb

```diff
@@ -19,6 +19,15 @@ class Notification < ActiveRecord::Base
 
   VALID_STATUSES = STATUS_STATES.keys
   validates :status, inclusion: { in: VALID_STATUSES }
+  validate :notification_has_at_least_one_recipient
+
+   def notification_has_at_least_one_recipient
```

Comment:
The indentation here is a little off, but more importantly - should this be a private method instead?

Reply:
Yes, good call